lib/config-builder.js

```diff
@@ -81,7 +81,8 @@ const commitMessage = {
 const shared = merge.all([
   {
     extends: [":disableRateLimiting"],
-    enabledManagers: ["npm", "circleci"],
+    enabledManagers: ["npm", "circleci", "bundler"],
+    bundler: { enabled: true },
```

Comment:
So this part is great 👍. 

We still need to make rules around how we go about updating gems. Right now there are no rules specifying how the ruby dependency updates should function and worse we've disabled things like rate limiting so it's potentially gonna open up a _bunch_ of PRs. 

Still, this is a great start!

Reply:
I kind of wanted to see how this vanilla setup looked before going into rules, unless you think it's a must from the start. Wondering about why the rate limiting is disabled, and also have some questions like:

With the master issue thing wouldn't there be just one PR created with the list of all deps?

Also, what kind of rules should we have about which gems to update, like a manual allow-list of updatable gems and a block-list of system gems like Rails, and that sort of stuff?

I'd be down to pair on that 👯‍♂

Reply:
Without rules it would just open 1 PR for every update that's available. Generally it would be rate limited, but we have that turned off because for the package managers we have available, we restrict their usage pretty heavily in other ways. 

Master issue approval is _only_ enabled for the manager `npm`. It functions by creating an [issue](https://github.com/artsy/force/issues/4025) in the repo with a checklist of updates. 

I don't really know what the rules for gems should be, but if you have a good idea of how you'd approach the problem manually then we can work backwards from there and come up with some good rules. 

Yeah, let's pair!